apps/web/src/components/Common/Layout.tsx

```diff
@@ -93,6 +97,14 @@ const Layout: FC<LayoutProps> = ({ children }) => {
         position="bottom-right"
         toastOptions={getToastOptions(resolvedTheme)}
       />
+      {transactions.map((tx) => (
+        <OaTransactionToaster
+          key={tx.txHash}
+          onClose={() => {}}
+          platformName={tx.platformName}
+          txHash={tx.txHash}
+        />
+      ))}
```

Comment:
Can we reuse this file

`apps/web/src/components/Shared/Modal/OptimisticTransactions/Transaction.tsx`

We don't want to show long pending toast, let it be optimistic! on mint we just show mint success